RELEASES.md

```diff
@@ -1,3 +1,14 @@
+Version 1.85.1 (2025-03-18)
+==========================
+
+<a id="1.85.1"></a>
+
+- [Fix the doctest-merging feature of the 2024 Edition.](https://github.com/rust-lang/rust/pull/137899/)
+- [Relax some `target_feature` checks when generating docs.](https://github.com/rust-lang/rust/pull/137632/)
+- [Fix errors in `std::fs::rename` on Windows 1607.](https://github.com/rust-lang/rust/pull/137528/)
```

Comment:
It doesn't matter much but it's been pointed out to me that this should rightly be "Windows 10, version 1607" to avoid confusing it with the year 1607. We don't officially support Windows versions that far back so there shouldn't be any real ambiguity but it does make sense to be accurate. But it's no big deal either way.

Reply:
> [...] to avoid confusing it with the year 1607. We don't officially support Windows versions that far back [...]

Hmm, is _that_ why the Window epoch starts in 1601? :thinking: